src/app/helptext/system/ssh-connections.ts

```diff
@@ -0,0 +1,38 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T(''),
+
+    setup_method_placeholder: T('Setup Method'),
+    setup_method_tooltip: T(''),
+
+    host_placeholder: T('Host'),
+    host_tooltip: T(''),
+
+    port_placeholder: T('Port'),
+    port_tooltip: T(''),
+
+    username_placeholder: T('Username'),
+    username_tooltip: T(''),
+
+    private_key_placeholder: T('Private Key'),
+    private_key_tooltip: T(''),
+
+    remote_host_key_placeholder: T('Remote Host Key'),
+    remote_host_key_tooltip: T(''),
+
+    cipher_placeholder: T('Cipher'),
+    cipher_tooltip: T(''),
+
+    connect_timeout_placeholder: T('Connect Timeout'),
+    connect_timeout_tooltip: T(''),
+
+    url_placeholder: T('FreeNAS URL'),
+    url_tooltip: T(''),
```

Comment:
From Guide Example: FreeNAS to FreeNAS Semi-Automatic Setup section

IP address or hostname of the remote FreeNAS system.

Reply:
It's not just IP/hostname anymore, it's URL with `http://` or `https://` at the beginning